Dockerfile

```diff
@@ -1,21 +1,30 @@
-from golang:1.11 as build
+from golang:1.12 as build
 
 ENV SRC github.com/segmentio/aws-okta
 ARG VERSION
 
-WORKDIR /build
-COPY . /go/src/${SRC}
-
 RUN apt-get update
 RUN apt-get install --no-install-recommends -y \
     libusb-1.0-0-dev \
     ca-certificates \
     build-essential \
     git
 
-RUN CGO_ENABLED=1 go build -o aws-okta -ldflags="-X main.version=$VERSION" ${SRC}/cmd
+WORKDIR /src
+COPY . .
+
+RUN go mod download 
+RUN CGO_ENABLED=1 GOOS=linux go build -o /aws-okta -ldflags="-X main.version=$VERSION"
+
+FROM golang:1.12 as debug
```

Comment:
I'm not sure I understand why this layer is here? The only artifacts are from `apt-get`?

Reply:
Yeah, it's a little weird. We use `debug` as the build target in the docker-compose.yml. I suppose a separate Dockerfile might be less confusing.

Reply:
I think there's a bunch of stuff that needs cleanup here, and I was trying to do too much at once, and it's clouded the main purpose of the PR